controllers/machinehealthcheck_controller_test.go

```diff
@@ -464,6 +468,28 @@ func TestReconcileExternalRemediationTemplate(t *testing.T) {
 				},
 			},
 		},
+
+		{
+			name:                "create new multiple template supported remediation",
+			machine:             machineWithNodeUnHealthy,
+			node:                nodeUnHealthy,
```

Comment:
can we please add a test for the remediation CR annotations, probably in `verifyRemediationCR`?